src/civis/io/_files.py

```diff
@@ -156,8 +151,6 @@ def _multipart_upload(buf, name, file_size, client, **kwargs):
     if num_parts != len(urls):
         raise ValueError(f"There are {num_parts} file parts but only {len(urls)} urls")
 
-    # upload function wrapped with a retry decorator
-    @_retry(RETRY_EXCEPTIONS)
```

Comment:
Switching to the default retryer. New unit test written as`test_file_multipart_upload_retries` below for this helper function.